apps/src/constants.js

```diff
@@ -26,6 +26,7 @@ exports.TestResults = {
   NO_TESTS_RUN: -1,
 
   // The level was not solved.
+  GENERIC_FAIL: 0,                    // Used by DSL defined levels.
```

Comment:
Again, I'm not suggesting that 0 is the right value - but it is the value used by multi/match, so I don't want to send a different value starting today. We can rename the constant to `NOT_TRIED_OR_DSL_LEVEL_FAIL`?